src/lib/options/options.js

```diff
@@ -10,32 +10,36 @@ export default class Options {
    * Mandatory fields:
    *    {string} domain - A domain name that defines options context
    *    {Object} items - An object that represents options that are exposed to the user. Each property is an option name.
-   * @param {Function<StorageAdapter>} StorageAdapter - A storage adapter implementation
+   * @param {StorageAdapter} storageAdapter - A storage adapter implementation
+   * @param {Object} adapterInitObj - an object which can be passed to the Storage Adapter to provide additional application state
```

Comment:
Has this been implemented?

Reply:
oh, no that was from an earlier approach that I decided against. Thanks I will remove.